### ISSUE TITLE
Problem with the window, could not show on Windows 8.1

### ISSUE THREAD

**Issue by aprilorange** [maxogden/menubar#32]
I cant see any window show up but only a tab with a preview. 

Is it only my problem?

![gif](http://i.gyazo.com/5e6f77b79e09b1bf192cf0b2dcae634e.gif)

**Comment by fabien-d** [comment]
Could you post sample code that reproduces the issue? We're currently developing an application for Windows and Mac and haven't had any issues.

**Comment by aprilorange** [comment]
@fabien-d have you tested it on Windows 8.1 ? I followed the REAMEDE,md and used exactly the example code.

**Comment by fabien-d** [comment]
I ran the example from the repo (on Windows 8.1) and it works properly. Could you confirm running if the example fails for you?

![image](https://cloud.githubusercontent.com/assets/747616/8626711/1974080c-2714-11e5-8878-d8e3e778ce0f.png)

**Comment by aprilorange** [comment]
@fabien-d eh, I retried the example on a my laptop with newly installed windows 7and failed again... :weary:

**Comment by fabien-d** [comment]
Are you running the example in the `example` directory or Monu (https://github.com/maxogden/monu)? The icon in the first screencast appears to be Monu which may be using an older version of menubar before Windows supported was added.

Windows support was added as of https://github.com/maxogden/menubar/pull/25.

**Comment by kevinpsiu** [comment]
Just to confirm that @aprilorange isn't crazy, I reproduced the bug using the latest menubar version straight from repo as of today. 

I did a bit of digging, and perhaps this snippet behaves differently on different machines:

```javascript
    function showWindow (trayPos) {
      var x = opts.x || Math.floor(trayPos.x - ((opts.width / 2) || 200) + (trayPos.width / 2))
      var y = opts.y || trayPos.y
      if (!menubar.window) {
        createWindow(true, x, y)
      }
    ...
```
The variables I logged gave me an `undefined` for `opts.y` and `1042` for `y` (which was `trayPos.y`). `trayPos.y` seemed to be the right value for the absolute position of the top of the tray, however, it did not account for the height of the window.

Upon some further reading, it appears to me that this line of code further up was not triggered:

    if (process.platform === 'win32') bounds.y = size.height - opts.height


because it was wrapped in 

    if (bounds.x === 0 && bounds.y === 0) 
 
When I moved the `if (process.platform) ...` line outside of that if statement, the window started to behave correctly. As I am not familiar with the code, I'm not sure if you intended to do that (any particular reason for the platform check to be done only if `bounds` is not set?). Otherwise, my suggestion would fix it (sorry for being verbose).

**Comment by fabien-d** [comment]
Any chance you could try and reproduce with the following branch? https://github.com/maxogden/menubar/pull/31.

> any particular reason for the platform check to be done only if bounds is not set?

The `bounds` object, when set, should contain the proper values and shouldn't need to be modified. However, it doesn't get set on all OS. The `x === 0 && y === 0` check confirmed if it was set or not.

**Comment by kevinpsiu** [comment]
Same bug reproduced in #31, but reading through the code, I think the logic there is closer. Not sure why it was reverted.

#31 can be fixed by editing the following:

      // default to bottom on windows if `bounds.y` is not set
      // leave it open to use prepopulated `bounds` if/when windows sets it
      if (process.platform === 'win32' && bounds.y === 0) bounds.y = size.height - opts.height

Should be 

      if (process.platform === 'win32' && bounds.y !== 0) bounds.y = size.height - opts.height

**Comment by fabien-d** [comment]
If you log the `bounds` object, what do you get? When I test on Windows 8.1, the `bounds` object doesn't get set (all values are `0`), which allows the `y === 0` test to be caught properly.

The `y === 0` is supposed to be a temporary solution so when Windows pre-populates the object, this check would be skipped. If checking `y !== 0`, this condition would overwrite the natively set `bounds` object. 

![image](https://cloud.githubusercontent.com/assets/747616/8802302/8dbc7104-2f8e-11e5-97bd-346e167777f6.png)

**Comment by fabien-d** [comment]
I've removed the `y` check completely in #31. It appears that even when the `bounds` get set, it doesn't get set properly (ignores the height of the app).

See https://github.com/maxogden/menubar/commit/ca33c376a92429eb8db4bbd8ca297f83ccb4df67.

Appreciate all the info, it's hard to debug when you can't reproduce!

**Comment by kevinpsiu** [comment]
That works. Cheers.

**Comment by fabien-d** [comment]
Resolved with the release of `2.0.17`
